content/docs/2.4/scalers/influxdb.md

```diff
@@ -30,18 +30,18 @@ triggers:
 
```

Comment:
Would you mind updating all our 2.0->2.3 docs to remove the authentication parameters there please?

Reply:
Sorry, I'm still not sure I'm following this. There has been no mention of TriggerAuthentication support for influxdb in any of the old version of the docs.

Reply:
No problem at all, I was just checking. Would you mind updating 2.5 as well please?